alignak/objects/notificationway.py

```diff
@@ -352,7 +385,7 @@ def new_inner_member(self, name=None, params=None):
         :return: None
         """
         if name is None:
-            name = NotificationWay._id
+            name = 'Generated_notificationway_%s' % uuid.uuid4().hex
```

Comment:
We are in Notificationways, no a Notificationway. Here we create a new object. Still there is a small mistake here in my opinion